packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -46,26 +49,33 @@ export const SideNavListItemContainer = styled("li", {
   "&:last-child": {
     marginBottom: odysseyDesignTokens.Spacing2,
   },
-  "& a": {
+  "& a, & div[role='button']": {
     display: "flex",
     alignItems: "center",
     width: "100%",
-    minHeight: "48px",
-    padding: `${odysseyDesignTokens.Spacing3} ${odysseyDesignTokens.Spacing4}`,
+    minHeight: isCompact ? "37px" : "48px",
```

Comment:
These heights are important; therefore, we should store them somewhere else so it's obvious what they are and where they're located:
```ts
export const minimumHeight = {
  compact: "37px",
  comfortable: "48px",
}
```

I think we should make these relative to `rem` instead of `px` right? If you change the HTML font-size, these won't be affected, but everything else will. I'm not sure that's what we want.